src/test/java/com/ably/kafka/connect/MessageSinkMappingTest.java

```diff
@@ -109,6 +110,7 @@ void testGetMessage_messageDataIsTheSameWithRecordValue() {
 
 
     @Test
+    @Disabled
```

Comment:
Can we add a `// TODO` so that we know to come back to this?

Reply:
Added ToDo, Its related to this change
```
        //ToDo: Temporarily disabling, needs to be enabled again for Idempotency.
        //message.id = String.format("%d:%d:%d", record.topic().hashCode(), record.kafkaPartition(), record.kafkaOffset());

```